content/source/docs/cloud/integrations/kubernetes/index.html.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: "cloud"
+page_title: "Terraform Cloud Operator for Kubernetes setup instructions"
+description: |-
+  HashiCorp Terraform Cloud customers can integrate with Kubernetes using the official Terraform Cloud Operator for Kubernetes to provision infrastructure through Terraform Cloud using their Kubernetes control plane
+---
+
+# Terraform Cloud Operator for Kubernetes Setup Instructions
+
+-> **Note:** The Terraform Cloud Operator for Kubernetes is still under development and in the alpha testing stage. It is not ready for production use.
+
+## Overview
+
+HashiCorp Terraform Cloud customers can integrate with Kubernetes using the official [Terraform Cloud Operator for Kubernetes](https://github.com/hashicorp/terraform-k8s) to provision infrastructure internal or external to the Kubernetes cluster directly from the Kubernetes control plane.  Using the Terraform Cloud Operator for Kubernetes' CustomResourceDefition (CRD), users can dynamically create Terraform Cloud workspaces, populate variables, and apply a Terraform configuration stored in a git repository or directly from the [Terraform Registry](https://registry.terraform.io/).
```

Comment:
- Last sentence was unclear; it probably meant you can use a registry module as the root module for a workspace, so I changed it to keep that associated with the right bit.
- Typo (defition).

```suggestion
HashiCorp Terraform Cloud customers can integrate with Kubernetes using the official [Terraform Cloud Operator for Kubernetes](https://github.com/hashicorp/terraform-k8s) to provision infrastructure internal or external to the Kubernetes cluster directly from the Kubernetes control plane.  Using the Terraform Cloud Operator for Kubernetes' CustomResourceDefinition (CRD), users can dynamically create Terraform Cloud workspaces (using a Terraform configuration from a git repository or from the [Terraform Registry](https://registry.terraform.io/), populate variables, and perform Terraform runs to provision infrastructure.
```